### DISCUSSION TITLE
Android Auto hiting the end of the road soon

### DISCUSSION THREAD

**Discussion by galosre** [tesla-android/issue-tracker#47] [General]
Soon going to be replaced by Google Assistant driving. Just wondering what is gonna happen with all the dongle ?

https://9to5google.com/2022/06/02/android-auto-for-phone-screens-shut-down/
https://www.androidpolice.com/android-auto-for-phone-screens-dead/
https://www.theverge.com/2022/6/3/23153100/android-auto-app-phone-screens-discontinued

**Comment by gltovar** [comment]
you misunderstand, this is the android auto app that turns your phone screen into a driving friendly interface, NOT the remote screen that external touchscreens display when connected with an android phone. Android Auto (the android equivalent to Apple Carplay)  will continue to exist for quite a while, I'd imagine.
